src/NServiceBus.NHibernate.AcceptanceTests-SqlTransportTests/NServiceBus.NHibernate.AcceptanceTests-SqlTransportTests.csproj

```diff
@@ -1,8 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net452</TargetFramework>
+    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
     <RootNamespace>NServiceBus.NHibernate.AcceptanceTests.SqlTransport</RootNamespace>
```

Comment:
This shouldn't be required. It would be fine to let the name derived from the project be the default.

Reply:
I left the namespace unchanged (project name has been changet to get picked up by build configs)

Reply:
The reason I was suggesting to remove the line is that if you look at all the files in the project, none of them are in the `NServiceBus.NHibernate.AcceptanceTests.SqlTransport` namespace, so there's no real reason to care about maintaining that value.

The new default value based on the project name is just as good, so it seems better to remove an unneeded line from the project file!